x-pack/plugins/lens/public/editor_frame_service/editor_frame/workspace_panel/warnings_popover.scss

```
@@ -8,6 +8,7 @@
   @include euiYScroll;
   max-height: $euiSize * 20;
   width: $euiSize * 16;
+  word-break: break-word;
```

Unless I'm mistaken, I believe the `break-word` value for the `word-break` CSS property is deprecated. Can we try the below suggestion instead?

```suggestion
  overflow-wrap: break-word;
```